index.js

```diff
@@ -5,9 +5,13 @@ exports.decompress = decompress;
 
 var brotli  = require('iltorb');
 var through = require('through2');
+var extend = require('extend');
 
 function compress(params) {
-  params = params || {};
+  params = extend(true, {
```

Comment:
True. I added `extend ` such that extra parameters can be added easily in the future. I will change this.